.gitignore

```diff
@@ -21,7 +21,7 @@ target
 /esvm
 .htpasswd
 .eslintcache
-plugins
+/plugins
```

Comment:
Yup, I only meant this one line, sorry if it sounded like I was implying we change the whole file